src/diff/amfGraphDifferencer.ts

```diff
@@ -184,6 +193,12 @@ function parseNodePropDelta(
   nodeType: string[],
   nodeDelta: AmfGraphDeltaTypes.NodePropertyDelta
 ): NodeChanges {
+  if (!isNodeTypeDefined(nodeType)) {
+    ramlToolLogger.debug(
+      `Ignoring changes to the node with no type: ${nodeId}`
+    );
+    return undefined;
```

Comment:
```suggestion
    return;
```
`return` implicitly returns `undefined`.

Reply:
we do not allow implicit returns -  "noImplicitReturns" is set true